rasa/server.py

```diff
@@ -1286,7 +1286,9 @@ async def parse(request: Request) -> HTTPResponse:
         try:
             data = emulator.normalise_request_json(request.json)
             try:
-                parsed_data = await app.agent.parse_message(data.get("text"))
+                parsed_data = await app.agent.parse_message(
+                    data.get("text"), data.get("metadata")
+                )
```

Comment:
The metadata should already form part of the standard `UserMessage` object that is passed all the way down to Duckling. Can you clarify why it's necessary to pass it in explicitly here?